tests/app/helpers.py

```diff
@@ -11,3 +11,13 @@ def setUp(self):
 
     def tearDown(self):
         self.s3.stop()
+
+
+class LoggedInApplicationTest(BaseApplicationTest):
+    def setUp(self):
+        super(LoggedInApplicationTest, self).setUp()
+        with self.client.session_transaction() as session:
+            session['username'] = 'admin'
+
+    def tearDown(self):
```

Comment:
We're not overriding the tearDown from `BaseApplicationTest`, so we can just remove it